src/syntree.rs

```diff
@@ -1,31 +1,40 @@
 use crate::compiler::Prec;
 use crate::error::Error;
 use crate::tokenizer::file_to_tokens;
-use crate::tokenizer::Token as T;
+use crate::tokenizer::Token;
 use crate::Next;
 use crate::Type as RuntimeType;
 use std::collections::{HashMap, HashSet};
 use std::path::{Path, PathBuf};
 
+type T = Token;
+
 #[derive(Debug, Copy, Clone)]
+/// A location in a file containing source code.
 pub struct Span {
     // TODO(ed): Do this more intelligent, so
     // we can show ranges. Maybe even go back
     // to offsets from start of the file.
     line: usize,
 }
 
+/// Contains modules.
 #[derive(Debug, Clone)]
 pub struct Prog {
     modules: Vec<(PathBuf, Module)>,
 }
 
+/// Contains statements.
 #[derive(Debug, Clone)]
 pub struct Module {
     span: Span,
     statements: Vec<Statement>,
 }
 
+/// Variables can be any combination of `{Force,}{Const,Mutable}`.
```

Comment:
```suggestion
/// Variables can be any combination of `{Force}{Const|Mutable}`.
```
Clearer? Or is this some rust thing I don't know about?

Reply:
Shell expansions.
```
$ cat {a,}{b,c}
cat: ab: No such file or directory
cat: ac: No such file or directory
cat: b: No such file or directory
cat: c: No such file or directory
```